_layouts/base.html.slim

```diff
@@ -93,6 +93,12 @@ body.antialiased
       - unless page.url.start_with? '/news/'
         .large-4.columns.secondary
           - if page.url =~ /^\/docs\/.+/
+            - if page.toc
```

Comment:
We deprecated `convert_with_options` in favor of a consolidated `convert` method, so that call should be:

```
document.converter.convert document, 'outline', toclevels: 3
```